utils/rpms/daos.spec

```diff
@@ -333,6 +335,9 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_libdir}/*.a
 
 %changelog
+* Fri Dec 13 2019 Jeff Olivier <jeffrey.v.olivier@intel.com> - 0.7.0-2
+- Remove openmpi, pmix, and hwloc build requirement, use packages
```

Comment:
Adding `hwloc`[`-devel`] build requirement here though, aren't we?

Reply:
I modified the comment